test/dashboard-e2e/data-handlers/test-data-handlers.ts

```diff
@@ -34,4 +35,12 @@ export class TestDataHandler {
 
         return testSuite;
     }
+
+    getExecutor(executorName) {
+        let executor = executorData[executorName]
+        executor.name = this.getRandomizedName(executor.name);
+        executor.types[0] = this.getRandomizedName(executor.types[0]);
+
+        return executor;
+    }
```

Comment:
That will return the same instance of object executor, probably not expected. It's better to make it immutable:

```suggestion
    getExecutor(executorName) {
        return {
            ...executorData[executorName],
            name: this.getRandomizedName(executor.name), 
            types: [ this.getRandomizedName(executor.types[0]), ...types.slice(1) ],
        };
    }
```